Adds command `pp dataverse table row list`, Closes #3791 (#4077)
pnp/cli-microsoft365: docs/docs/cmd/pp/dataverse/dataverse-table-row-list.md

# pp dataverse table row list

> Lists dataverse table rows in a given environment

Reviewer: ```suggestion
Lists table rows for the given Dataverse table
```

# pp dataverse table row list

Lists table rows for the given Dataverse table

## Usage

```sh
pp dataverse table row list [options]
```

## Options

`-e, --environment <environment>`
: The name of the environment

`-n, --entitySetName [entitySetName]`
: The entity set name of the table. Specify either `entitySetName` or `tableName` but not both

`-t, --tableName [tableName]`
: The name of the table. Specify either `entitySetName` or `tableName` but not both

`--asAdmin`
: Run the command as admin for environments you do not have explicitly assigned permissions to

--8<-- "docs/cmd/_global.md"

## Examples

List all table rows for the given environment based on entitySetName parameter

```sh

> m365 pp dataverse table row list -e "Default-2ca3eaa5-140f-4175-8261-3272edf9f339" --entitySetName "cr6c3_accounts"

Reviewer: Let's not use short options in our examples.
